src/content/docs/apis/nerdgraph/examples/configure-infinite-tracing-graphql.mdx

```diff
@@ -159,6 +159,236 @@ Here's the response confirming the change:
 }
 ```
 
+## Update the data sources on a trace observer [#updating-the-data-sources]
+
+While some entities can be configured to send their tracing data directly to an Infinite Tracing trace observer, others can have their data routed by New Relic to a trace observer. Currently, we support this feature for mobile, browser, and Lambda entities. In the context of Infinite Tracing, we call these entities data sources. 
+
+<Callout variant="tip">
+  If you need help about data sources in the trace observer, see [Set up the trace observer](/docs/distributed-tracing/infinite-tracing/set-trace-observer/#browser-mobile-lambda).
+</Callout>
+
+There are three ways you can update the data sources on a trace observer:
+
+* **ADD:** If you'd like to add new data sources to the existing ones on a trace observer, use the `ADD` `DataSourceGroupUpdateType`.  No data sources will be removed with this mutation.
+* **REPLACE:** If you'd like to replace your existing data sources on a trace observer with a new set of values, use the `REPLACE` `DataSourceGroupUpdateType`. Any existing values not in the new set will have their status set to `INACTIVE`.
+* **REMOVE:** If you'd like to remove one or more of your existing data sources on a trace observer, use the `REMOVE` `DataSourceGroupUpdateType`. Any values passed will have their status set to `INACTIVE`.
+
+The following example shows you how to add to the existing data sources:
+
+1. Go to the NerdGraph GraphiQL explorer at [api.newrelic.com/graphiql](https://api.newrelic.com/graphiql).
+2. Execute the following query to find the trace observer that contains the data source group you would like to modify:
+
+   ```
+  {
+    actor {
+      account(id: <var>YOUR_ACCOUNT_ID</var>) {
+        edge {
+          tracing {
+            traceObservers {
+              id
+              name
+              dataSourceGroup {
+                dataSources {
+                  entity {
+                    guid
+                    name
+                    entityType
+                  }
+                  status
+                }
+              }
+              endpoints {
+                agent {
+                  host
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+   ```
+3. In the response, find the trace observer `id`. Here is an example with two trace observers. Let's say you want the one in US-EAST-1 called "Production services" where the `id` value is `259`:
```

Comment:
I'd go: "Let's say you want the one in US-EAST-1 called "Production services", whose `id` is `259`:"

Reply:
@urbiz-nr, thanks! I'll use this!